src/services/TestDataService.ts

```diff
@@ -97,7 +100,7 @@ export class TestDataService {
      *
      * @private
      */
-    private highPriorityEntities = ['order', 'product', 'landing_page', 'sales_channel_currency', 'sales_channel_country'];
+    private highPriorityEntities = ['order', 'product', 'landing_page', 'sales_channel_currency', 'sales_channel_country', 'sales_channel_language'];
```

Comment:
I figured out that we have to add 'customer' here and delete 'sales_channel_language' because during the deletion of a customer with a assigned language, the language has to be present in the sales channel as long the customer is available with that specific language.